src/components/RichTextEditor/plugins/linkPlugin/FloatingLink/CustomFloatingLink.tsx

```diff
@@ -0,0 +1,100 @@
+import { Button, ButtonSize, ButtonStyle } from '@components/Button/Button';
+import { LINK_CLASSES } from '@components/RichTextEditor/components';
+import { IconCheckMark, IconPen, IconTrashBin } from '@foundation/Icon';
+import {
+    floatingLinkActions,
+    submitFloatingLink,
+    useEditorRef,
+    useFloatingLinkSelectors,
+    useFloatingLinkUrlInput,
+} from '@udecode/plate';
+import React, { useState } from 'react';
+import { FloatingLink } from './FloatingLink';
+
+export const CustomFloatingLink = () => {
+    const isEditing = useFloatingLinkSelectors().isEditing();
+    const editor = useEditorRef();
+    const [isUrlInValid, setIsUrlInValid] = useState(false);
```

Comment:
You mean invalid, right? I would write it than with a small `v` like: `isUrlInvalid` .